src/services/administration/iam/policy/policy-create/PolicyCreatePage.vue

```diff
@@ -66,11 +78,33 @@ export default {
     },
     setup() {
         const state = reactive({
-            code: '{}',
+            name: '',
+            code: '{"permissions": ["*"]}',
         });
 
+        const handleNameInput = (inputtedName: string) => { state.name = inputtedName; };
+        const handleCreatePolicy = () => {
+            try {
+                SpaceConnector.client.identity.policy.create({ ...JSON.parse(state.code), name: state.name })
+                    .then(() => {
+                        // song-lang
+                        showSuccessMessage('Successfully created policy', '');
+                        SpaceRouter.router.go(-1);
+                    });
+            } catch (e: any) {
+                // song-lang
+                showErrorMessage('Failed to create policy', e);
+                throw new Error(e);
+            }
+        };
```

Comment:
```suggestion
        const handleCreatePolicy = async () => {
            try {
                await SpaceConnector.client.identity.policy.create({ ...JSON.parse(state.code), name: state.name })

                // song-lang
                showSuccessMessage('Successfully created policy', '');
                SpaceRouter.router.go(-1);

            } catch (e: any) {
                // song-lang
                ErrorHandler.handleRequestError(...)
                throw e;
            }
        };
```

기본적으로 async await 문법을 사용하는 것이 컨벤션입니다!

Reply:
showErrorMessage를 직접 호출하지 않고, ErrorHandler를 사용하여야 일관된 방식으로 에러에 대처 가능합니다!